tests/features/sitemap.feature

```diff
@@ -4,11 +4,20 @@ Feature: Check the Sitemap
   I can publish a sitemap
 
   @api
+  Scenario: Value of the variable xmlsitemap_prefetch_aliases is zero
+    Given the module is enabled
+      | modules     |
+      | sitemap     |
+    Given I am an administrator user
```

Comment:
Hi,
Maybe we can use 
Given I am logged in as a user with the "administrator" role

Reply:
Indeed, we have 2 different steps for this, we should only have one to be more consistent